_config.yml

```diff
@@ -180,6 +180,9 @@ footerBlocks:
   title: "FAQ"
   links:
    - {permalink: "/code-of-conduct/", text: "Code of Conduct"}
+   - {permalink: "/diversity-inclusion/", text: "Diversity Inclusion"}
```

Comment:
I think this should be "Diversity & Inclusion" (the ampersand might need to be escaped; I'm not sure.)